docs/host_for_multiuser/setup.md

```diff
@@ -62,90 +64,97 @@ Follow the steps below to setup `multiuser` mode.
     - (keep databaseURL blank)
 7. Select "Service accounts" tab.
 8. Click "Generate new private key" in "Firebase Admin SDK" section.
-9. Put the downloaded file to `studio/config/auth/firebase_private.json`.
-
-### Setup mysql(or mariadb)
-1. Install mysql(or mariadb) server.
-2. Connect mysql server.
+9. Save the downloaded file to `studio/config/auth/firebase_private.json`.
+
+### Setup Database
+- Set up your own mysql (or mariadb) server or use docker compose mysql
+- Below are the instructions for using mysql with docker compose.
+
+1. Edit configs.
+    - Edit studio/config/.env
+      - Set `MYSQL_SERVER` to db server host or ip
+        - \*For docker platform, the fixed value `db:3306` is fine.
+      - Set `MYSQL_ROOT_PASSWORD` to {DB_ROOT_PASSWORD}, which you have decided.
+      - Set `MYSQL_DATABASE` to {YOUR_DATABASE_NAME}, which you have decided.
+      - Set `MYSQL_USER` to {DB_USER_NAME}, which you have decided.
+      - Set `MYSQL_PASSWORD` to {DB_USER_PASSWORD}, which you have decided.
+2. Install & run mysql server.
     ```bash
-    mysql -u root -p
-    ```
-3. Create database for your project.
-    ```sql
-    CREATE DATABASE YOUR_DATABASE_NAME;
+    docker compose -f docker-compose.dev.multiuser.yml up db -d
     ```
-4. Create user for your project.
-    ```sql
-    CREATE USER 'DB_USER_NAME'@'localhost' IDENTIFIED BY 'DB_USER_PASSWORD';
-    ```
-5. Grant all privileges to the user for the database.
-    ```sql
-    GRANT ALL PRIVILEGES ON YOUR_DATABASE_NAME.* TO 'DB_USER_NAME'@'localhost';
+    - The database and db_user are automatically generated based on the .env settings.
+3. Connect to mysql server.
+    ```bash
+    docker exec -it {DB_DOCKER_CONTAINER_NAME} mysql -u {DB_USER_NAME} -p {YOUR_DATABASE_NAME}
     ```
+    - If a connection to the database server is available, the setup was successful.
 
-### Set OptiNiSt config
-1. Edit `studio/config/.env`
+### Setup & Run OptiNiSt
+
+#### For Docker Platform
+
+##### Setup Backend
+- See [OptiNiSt installation guide](../installation/docker_for_developer.md).
```

Comment:
Personally, I don't see any problem (for initial construction of the environment),
Shall I use `cp -i` to display the confirm dialog?

Reply:
There is no problem with installation/docker_for_developer.md

The probelm is this line
> See [OptiNiSt installation guide](https://github.com/arayabrain/barebone-studio/installation/docker_for_developer.html).

Can we delete this line? It is confusing to have it here. Is there any benefit having it here?  

Having a link to another page here may be confusing because people may follow the link (as i did) and perform the steps on the other page first, and then proceed with steps of `docker compose -f docker-compose.dev.multiuser.yml up studio-dev-be -d`. This cause me errors.

Reply:
> There is no problem with installation/docker_for_developer.md

Removed the section above. Please recheck.